common/test/acceptance/pages/studio/settings_certificates.py

```diff
@@ -312,59 +351,108 @@ def click_create_certificate_button(self):
         """
         Create a new certificate.
         """
-        disable_animations(self.page)
-        self.find_css('.action-primary').first.click()
-        self.page.wait_for_ajax()
+        disable_animations(self)
+        self.q(css='.action-primary').first.click()
+        self.wait_for_ajax()
 
     def click_save_certificate_button(self):
         """
         Save certificate.
         """
-        self.find_css('.action-primary').first.click()
-        self.page.wait_for_ajax()
+        self.q(css='.action-primary').first.click()
+        self.wait_for_ajax()
 
     def click_add_signatory_button(self):
         """
         Add signatory to certificate
         """
-        self.find_css('.action-add-signatory').first.click()
+        self.q(css='.action-add-signatory').first.click()
 
     def click_edit_certificate_button(self):
         """
         Open editing view for the certificate.
         """
-        self.find_css('.action-edit .edit').first.click()
+        self.q(css='.action-edit .edit').first.click()
 
     def click_cancel_edit_certificate(self):
         """
         Cancel certificate editing.
         """
-        self.find_css('.action-secondary').first.click()
+        self.q(css='.action-secondary').first.click()
 
     def click_certificate_details_toggle(self):
         """
         Expand/collapse certificate configuration.
         """
-        self.find_css('a.detail-toggle').first.click()
+        self.q(css='a.detail-toggle').first.click()
 
     def click_delete_certificate_button(self):
         """
         Remove the first (possibly the only) certificate from the set
         """
         self.wait_for_certificate_delete_button()
-        self.find_css('.actions .delete.action-icon').first.click()
+        self.q(css='.actions .delete.action-icon').first.click()
 
 
-class Signatory(object):
+class SignatorySectionPage(CertificatesPage):
     """
-    Signatory page object wrapper
+    SignatorySectionPage is the signatory section within CertificatesSection, There might be multiple signatories
+    in a certificate section so this section object can be used to used to identify unique section and apply
+    operations on it.
     """
     def __init__(self, certificate, prefix, mode, index):
+        """
+        Initialize SignatorySection Page
+
+        :param certificate: Container Section Page Object of the Signatory section
+        :param prefix: css selector of the container element
+        :param index: index of section in the signatory list on the page
+        :param mode: 'details' or 'edit', showing whether signatory is being displayed or edited
+
+        :return:
+        """
         self.certificate = certificate
```

Comment:
I'm not sure if you'd still need this declaration. Since you're now on an actual PageObject, I'm wondering if you are using some redundant code in here...wouldn't you just use self instead of self.certificate for things you'd want to do on the page?